### PR TITLE
RK-7980 - Imrproving the dotnet copy

### PR DESCRIPTION
**Pull request by LiranLast**

### CODE REVIEW

File: docs/dotnet-setup.md [hunk 1]
```diff
@@ -132,29 +114,17 @@ Windows: Simply run rookout_test_core_2.x.bat or rookout_test_core_3.x.bat respe
...
-Pdb (Program Data Base) file is a repository to maintain information about your application. 
-
-It stores the important debug information such as:
-* file names 
-* variable names 
-* offsets of the functions
-* file hashes
-
-Although Rookout doesn't run your program in debug mode, we need the PDB file to properly collect the data per your request.
+Rookout requires your application to be built and deployed with a debug information in the form of a pdb file. Rookout supports the `full`, `portable` and `pdbonly` configurations, while `embedded` is *not* supported.
```
Comment (ElDuderinos) [inline docs/dotnet-setup.md:117]: You removed the instructions on how to do this:
`<DebugType>full/portable/pdbonly</DebugType>`
If this is needed then we don't expect the customer to start googling how to do that...

**Review by ElDuderinos — CHANGES_REQUESTED** [review]

File: docs/dotnet-setup.md (hunk 1 above)
Comment (LiranLast) [reply docs/dotnet-setup.md:117]: That was hardly instructions :)

Where in the file does it go? Is it per project? Per solution? Something else? :)

Comment (ElDuderinos) [reply docs/dotnet-setup.md:117]: The same place where you place the Optimize, MSBuildGitHashCommand and EmbedAllSources parameters. Per project's PropertyGroup.

Comment (LiranLast) [reply docs/dotnet-setup.md:117]: That's not documented anywhere...

Want to suggest some copy? Maybe in a new section?

Comment (LiranLast) [reply docs/dotnet-setup.md:117]: @ElDuderinos can we merge?

**Comment by LiranLast** [comment]
@ElDuderinos let's merge this :)

**Review by ElDuderinos — APPROVED** [review]

**Pull request merged**